src/imdb.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# coding: utf8
+
+"""
+EPAM python q4 homework 05 2
+Sergey Streltsov 2019-11-17
+"""
+
+from operator import itemgetter
+import os
+import sys
+
+
+def load_ratings(filename):
+    run_path = os.path.dirname(os.path.abspath(__file__))
+    full_path = os.path.join(run_path, filename)
+    if os.path.isfile(full_path):
+        top250 = list()
+        is_top250 = False
+        is_header = False
+        with open(full_path, 'r', encoding='cp1252') as in_file:
+            for idx, line in enumerate(in_file):
```

Comment:
are you sure this enumerate wouldn't cause all file read? it looks to be risky from performance point of view

Reply:
agree